File: Semestre 2026.2/Programação de Computadores 2/Java/tabuada.java
import java.util.Scanner;

public class tabuada {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.println("=== Tabuada dinâmica ==");
        int opcao = 1;

        while(opcao == 1){
            System.out.println("Vamos lá, digite aqui o número o qual você deseja ver a tabuada: ");
            int numero = scanner.nextInt();

            System.out.println("=== Tabuada do " + numero + ": ===");

            for (int i = 1; i < 11; i++){
                System.err.println(numero + " x " + i + " = " + (numero*i));
            }

            System.out.println("Você deseja continuar? 1- sim | 2- não.");
            opcao = scanner.nextInt();
            if (opcao != 1){
                System.out.println("Fim do programa, volte sempre!");
                return;
            }

        }
        scanner.close();
    }
}

File: Semestre 2026.4/Logica/listas/ExerciciosBasicos/Tabuada.c
#include <stdio.h>

int main(){
    int num, contador, resultado;
    printf("=== Tabuada ===\n");
    printf("Digite qual o número que você quer ver a tabuada: ");
    scanf("%d", &num);

    contador = 1;
    printf("\n=== Tabuada do %d ===", num);
    for(int i = 1; i <= 10; i++){
        resultado = num * contador;
        printf("\n%dx%d=%d", num, contador, resultado);
        contador++;
    }
    printf("\n=== fim ===\n");
    //concluído...
    return 0;
}